internal/accounts/account-add.go

```diff
@@ -0,0 +1,102 @@
+package accounts
+
+import (
+	"bytes"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/onflow/flow-cli/internal/command"
+	"github.com/onflow/flow-cli/pkg/flow"
+	"github.com/onflow/flow-cli/pkg/flow/services"
+	"github.com/psiemens/sconfig"
+	"github.com/spf13/cobra"
+)
+
+type flagsAdd struct {
+	SigAlgo    string `default:"ECDSA_P256" flag:"algo,a" info:"Signature algorithm"`
+	HashAlgo   string `default:"SHA3_256" flag:"service-hash-algo" info:"Hashing algorithm"`
+	KeyIndex   int    `flag:"index" info:"Account key index"`
+	KeyHex     string `flag:"privateKey" info:"Private key in hex format"`
+	KeyContext string `flag:"context" info:"Projects/<PROJECTID>/locations/<LOCATION>/keyRings/<KEYRINGID>/cryptoKeys/<KEYID>/cryptoKeyVersions/<KEYVERSION>"`
```

Comment:
I agree, this sample was there from before, I would like if you can better explain the concept of Context, you have said it will be arbitrary data around the key itself, but still it can't just be completely arbitrary as we need to parse it, so how do we handle the input? With different supported formats as options or?